scripts/smtk2ssrf-build.sh

```diff
@@ -98,18 +110,55 @@ else
 	echo "----> Mdbtools already installed: $(pkg-config --print-provides libmdb)"
 fi
 
-# We are done. Move on
-# No need to update subsurface sources, as it has been previously
-# done by build.sh
-#
-# Do we want to build a branch or commit other than the one used
-# for build.sh?
-#
+# Build bare metal Subsurface.
+# We are going to modify some of the building parameters of Subsurface so
+# will get a copy of the cmake cache to restore them after building smtk2ssrf.
+cd "$SSRF_PATH" || aborting "Couldn't cd into $SSRF_PATH"
+echo "----> Saving a copy of $SSRF_PATH/build/CMakeCache.txt"
+cp -vf "$SSRF_PATH/build/CMakeCache.txt" "$SSRF_PATH/build/CMakeCache.txt.bak"
 if [ ! "$SSRF_TAG" == "" ]; then
-	cd "$SSRF_PATH" || aborting "Couldn't cd into $SSRF_PATH"
-	git checkout "$SSRF_TAG" || aborting "Couldn't checkout $SSRF_TAG. Is it correct?"
+	PREV_GIT="$(git branch --no-color 2> /dev/null | sed -e '/^[^*]/d')"
+	PREV_GIT=${PREV_GIT##*\ }; PREV_GIT=${PREV_GIT%)}
+	git checkout "$SSRF_TAG" || STATUS=1
+fi
+
+# abort if git checkout failed
+if [ ! -z "$STATUS" ] && [ "$STATUS" -eq 1 ]; then
+	mv -f "$SSRF_PATH/build/CMakeCache.txt.bak" "$SSRF_PATH/build/CMakeCache.txt"
+	aborting "Couldn't checkout $SSRF_TAG. Is it correct?"
 fi
 
+cmake   -DBTSUPPORT=OFF \
+	-DCMAKE_BUILD_TYPE="$RELEASE" \
+	-DFBSUPPORT=OFF \
+	-DFORCE_LIBSSH=OFF \
+	-DFTDISUPPORT=OFF \
+	-DMAKE_TESTS=OFF \
+	-DNO_DOCS=ON \
+	-DNO_MARBLE=ON \
```

Comment:
we no longer have the `NO_MARBLE` option. Subsurface is always build with the Map widget (from QtLocation).